src/script/interpreter.cpp

```diff
@@ -616,7 +616,32 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                     break;
                 }
 
-                case OP_NOP1: case OP_NOP4: case OP_NOP5:
+                case OP_CHECKTEMPLATEVERIFY:
+                {
+                    // if flags not enabled; treat as a NOP4
+                    if (!(flags & SCRIPT_VERIFY_STANDARD_TEMPLATE)) break;
```

Comment:
If the flag is not  enabled, we currently do not treat it as a NOP4 (e.g., discouraged). Will patch this shortly.

Reply:
referencing https://github.com/bitcoin/bitcoin/pull/6124/files it looks like it would be proper to discourage historically.